packages/foreman/foreman/foreman.spec

```diff
@@ -56,7 +56,7 @@ Requires(preun): systemd-units
 Requires: %{name}-debug
 
 # start specfile main Requires
-Requires: %{?scl_prefix}rubygem(rails) = 5.2.1
+Requires: %{?scl_prefix}rubygem(rails) = 6.0.2.1
```

Comment:
This needs a rev bump, but should probably not be done in this PR - https://github.com/theforeman/foreman-packaging/pull/5115 already updated this and other dependencies